docs/index.md

```diff
@@ -81,7 +81,7 @@ Download the latest version of the CLI for your operating system and make sure t
 
     === "Windows"
         ```sh
-        event-cli.exe help
+        pulse-event-cli.exe help
```

Comment:
I noticed that we weren't consistently using the same name for the Pulse CLI binary and I also fixed this both in the example and in the main `index.md` file.